### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage_Keyboard : All buttons such as "SQL Server" under the "SERVICE NAME" are not accessible through the keyboard

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7663]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
**GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG2.1.1;DesktopApp;#Keyboard;#SQL Azure Data Studio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name : Keyboard

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” view let (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Navigate till "troubleshoot " button.
10. Observe that All buttons such as "SQL Server" under the "SERVICE NAME" are not accessible through the keyboards

**Actual:**
All buttons such as "SQL Server" under the "SERVICE NAME" are not accessible through the keyboards.
Similar issue is observed for all buttons showed in image:
![image](https://user-images.githubusercontent.com/54656883/66642801-0b667380-ec3b-11e9-9816-693646dc4ea5.png)
![image](https://user-images.githubusercontent.com/54656883/66642813-11f4eb00-ec3b-11e9-8aa4-22b480dfd75d.png)

**Expected:**
All buttons such as "SQL Server" under the "SERVICE NAME" should be  accessible through the keyboards because they are all interactive.

**User Impact:**
Keyboard only users won't be able to navigate all the buttons.
**Attachment for Reference:**
[7663_A11y_AzureDataStudio_BigDataClusterOverviewPage_Keyboard .pptx](https://github.com/microsoft/azuredatastudio/files/3716817/7663_A11y_AzureDataStudio_BigDataClusterOverviewPage_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Duplicate of #7816

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
[7663_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4086058/7663_Closed.pptx)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev1

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
